bigtable-dataflow-parent/bigtable-beam-import/src/test/java/com/google/cloud/bigtable/beam/hbasesnapshots/HBaseSnapshotInputConfigBuilderTest.java

```diff
@@ -45,4 +46,63 @@ public void testBuildingHBaseSnapshotInputConfigBuilder() {
         conf.getClass(
             "mapreduce.job.inputformat.class", TableSnapshotInputFormat.class, InputFormat.class));
   }
+
+  @Test
+  public void testInvalidSnapshotHBaseSnapshotInputConfig() {
+    try {
+      new HBaseSnapshotInputConfigBuilder()
+          .setProjectId(TEST_PROJECT)
+          .setHbaseSnapshotSourceDir(TEST_SNAPSHOT_DIR)
+          .build();
+      fail();
```

Comment:
please add an error message in here: `fail("expected empty snapshot name to fail")` or something like that (and for all other assertions)